lib/universal-search.js

```diff
@@ -88,14 +88,27 @@ Search.prototype = {
   _loadStyleSheets: function(win) {
     const doc = win.document;
     const docEl = doc.documentElement;
+    //check whether platform type is Darwin, Linux, or WINNT
```

Comment:
We try to make comments complete sentences that start with a capital letter and end with punctuation.